modules/channel-slack/src/backend/index.ts

```diff
@@ -0,0 +1,58 @@
+import * as sdk from 'botpress/sdk'
+
+import { Config } from '../config'
+
+import { setupMiddleware, SlackClient } from './client'
+import { Clients } from './typings'
+
+let router
+const clients: Clients = {}
+
+const onServerStarted = async (bp: typeof sdk) => {
+  await setupMiddleware(bp, clients)
+}
+
+const onServerReady = async (bp: typeof sdk) => {
+  router = bp.http.createRouterForBot('channel-slack', {
+    checkAuthentication: false,
+    enableJsonBodyParser: false
```

Comment:
I'd add a second one here for `urlEncoded` not to break existing 3rd-party routers (see comment below)